src/config.ts

```diff
@@ -100,6 +102,7 @@ class Config {
      * @param data - The data from extension workspace settings to update the global configuration with.
      */
     updateConfig(rhdaConfig: any) {
+        this.exhortProxyUrl = rhdaConfig.proxyUrl || '';
```

Comment:
`rhdaConfig` here is for the most part the object associated with the `redHatDependencyAnalytics` config key from the vscode side. See [here](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/blob/master/src/server.ts#L95-L97) how its done. We don't have a `proxyUrl` under that key, where were you thinking that this is coming from?

Reply:
I thought it was the object passed by the vscode extension. I'll review that part. Thanks